server/src/weaverbird/backends/mongo_translator/steps/todate.py

```diff
@@ -7,13 +7,20 @@ def translate_todate(step: ToDateStep) -> list[MongoStep]:
 
     col_as_string = {"$toString": f"${step.column}"}
 
+    steps: list[dict] = []
+    should_guess_year = False
```

Comment:
```suggestion
    should_guess_century = False
```

We're guessing the century, right ?